catalog/catalog_products/views.py

```diff
@@ -0,0 +1,105 @@
+from typing import Dict
+from django.http.response import JsonResponse
+from rest_framework.generics import CreateAPIView
+from rest_framework.views import APIView
+from catalog_products.serializers import CategorySerializer, ProductSerializer
+from catalog_products.models import Category, Product
+from rest_framework import viewsets
+
+
+class CategoryAllViewSet(viewsets.ModelViewSet):
+    serializer_class = CategorySerializer
+    queryset = Category.objects.all()
+
+
+class ProductAllViewSet(viewsets.ModelViewSet):
+    serializer_class = ProductSerializer
+    queryset = Product.objects.all()
+
+
+class Task1ViewSet(APIView):
+    queryset = Category.objects.all()
+    serializer_class = CategorySerializer
+    def get(self, request):
+        category_all = Category.objects.all()
+        category_without_parent = []
+        category_with_parent = []
+        for category in category_all:
+            if category.parent != None:
+                category_with_parent.append(category)
+            else:
+                category_without_parent.append(category)
+        category_name_with_parent = list(map(lambda category: f'Subcategory: {category.name}, parent category: {category.parent.name}',category_with_parent))
```

Comment:
Please split these lines, it's hard to read them, also please install flake8 package and check the entire codebase for a pep8 violations